src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+        action="#"
+        method="post"
+        name="bigSearchBar"
+        data-qa="big"
+        class="bigSearch"
+      >
```

Comment:
2 tabs for indentation inside element for props

```suggestion
    <form 
      action="#"
      method="post"
      name="bigSearchBar"
      data-qa="big"
      class="bigSearch"
    >
```